### PR TITLE
Add tag prefix

### PR DESCRIPTION
**Pull request by buresmi7**
Hi, now is code from separated branch and immutable from other changes. Sorry for previous confusing changes.

### CODE REVIEW

File: .github/workflows/push.yml [hunk 1]
```diff
@@ -23,6 +23,8 @@ jobs:
           node-version: 8.10.0
       - name: "Automated Version Bump"
         uses: "phips28/gh-action-bump-version@master"
+        wiht:
```
Comment (phips28) [inline .github/workflows/push.yml]: typo

File: index.js [hunk 2]
```diff
@@ -45,6 +48,7 @@ Toolkit.run(async tools => {
       ['version', '--allow-same-version=true', '--git-tag-version=false', current])
     console.log('current:', current, '/', 'version:', version)
     newVersion = execSync(`npm version --git-tag-version=false ${version}`).toString().trim()
+    newVersion = `${tagPrefix}${newVersion}`
```
Comment (phips28) [inline index.js]: use:
```
newVersion = `${process.env['INPUT_TAG-PREFIX']}${newVersion}`
```
and remove `@actions/core` from dependency => speed

File: .github/workflows/push.yml (hunk 1 above)
Comment (buresmi7) [reply .github/workflows/push.yml]: fixed

File: index.js (hunk 2 above)
Comment (buresmi7) [reply index.js]: fixed

**Comment by phips28** [comment]
@buresmi7 thx :)

**Pull request merged**